resources/schema.gql

```diff
@@ -34,17 +34,17 @@ type Account {
 	isAdmin: Boolean!
 }
 
+enum AccountCategory {
```

Comment:
Don't really like the word "category" for this - in my head category is a many-to-many relationship like "paying" "free-trial" "non-paying" users.

Why did you decide to rename it?

Reply:
I do not like the word "type" even more. Trying to find a better term.

Reply:
what about "kind"?

Reply:
Reverting to "type". It seems many system use this "termless" term, including Github:
https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts